### PR TITLE
Tools paywall section refactor

### PR DESCRIPTION
**Pull request by wonderwomancode**
Moved "Adding a lock to a page" to Tutorials. Consolidated the rest of the subdirectory into one page. Improved the page layout.

### CODE REVIEW

File: docs/tools/paywall.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+---
+title: Paywall
+description: >-
+  Guide to the Paywall JavaScript library.
+---
+# Paywall
+
+The Paywall is a lightweight JavaScript library which can track state and emits
```
Comment (julien51) [inline docs/tools/paywall.md:8]: It's not that lightweight but we need to make it lighter :D

File: docs/tools/paywall.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+a the Checkout for purchasing keys.
+
+## Configuration
+
+The Paywall shares a configuration object with the Checkout and you can find
+everything you need to know in [Checkout / Configuration](/tools/checkout/configuration#the-paywallconfig-object) about how to build
+out the JSON object you're going use.
+
+## Events
+Once loaded the script will trigger events on the page’s ​window​ object.
```
Comment (julien51) [inline docs/tools/paywall.md:19]:
```suggestion

Once loaded the script will trigger events on the page’s ​window​ object. 
```

File: docs/tools/paywall.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+## Events
+Once loaded the script will trigger events on the page’s ​window​ object. 
+
+Registering event listeners.
+
+```javascript
+window.addEventListener("unlockProtocol.eventName", handler);
+```
+### Status
```
Comment (julien51) [inline docs/tools/paywall.md:26]:
```suggestion

### Status

```

**Review by julien51 — APPROVED** [review]
Let's fix the build but then LG

File: docs/tools/paywall.md (hunk 1 above)
Comment (wonderwomancode) [reply docs/tools/paywall.md]: Changed it to simple, which is true now :-)

**Pull request merged**